src/Compute/Compute/help/Invoke-AzVmPatchAssessment.md

```diff
@@ -0,0 +1,177 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Compute.dll-Help.xml
+Module Name: Az.Compute
+schema: 2.0.0
+---
+
+# Invoke-AzVMPatchAssessment
```

Comment:
Please make sure you regenerate help docs of the whole module, because `Az.Compute.md` is also supposed to be updated. 
See https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module